packages/client/src/i18nHelpers.ts

```diff
@@ -15,3 +16,37 @@ export function stepsFor (
       : i <= completed ? 'completed' : 'pending'
   }))
 }
+
+export function localizedError (
+  transfer: Transfer,
+  errorKey: string
+): string {
+  const type = getTransferType(transfer)
+  const i18n = type.i18n
+
+  // get user's locale from browser
+  const locale = navigator.language.replace('-', '_')
+
+  // get available & fallback locale from given i18n
+  const availableLocales = Object.keys(i18n)
+  const fallback = availableLocales[0] as string
```

Comment:
Should probably fall back to something that matches the first part of `locale` here. So if no `es_MX` but there is an `es_ES`, use `es_ES`.